blurb

```diff
@@ -650,24 +650,36 @@ Run unit tests.  Only works inside source repo, not when installed.
     print(tests_run, "tests passed.")
 
 
+def find_editor():
+    for var in 'GIT_EDITOR', 'EDITOR':
+        editor = os.environ.get(var)
+        if editor is not None:
+            return editor
+    if sys.platform == 'win32':
+        fallbacks = ['notepad.exe']
+    else:
+        fallbacks = [
+            '/etc/alternatives/editor',
+            '/usr/bin/editor',
+            'editor',
+            'nano',
+        ]
+    for fallback in fallbacks:
+        if os.path.isfile(fallback):
+            return fallback
+        found_path = shutil.which(fallback)
```

Comment:
This will do a which search for "/etc/alternatives/editor".

What I had in mind:

    if os.path.isabs(fallback):
        found_path = fallback
    else:
        found_path = shutil.which(fallback)
    if found_path and os.path.exists(found_path):
        return found_path
    error('...')